{
    "exceptions": {
        "child_service_not_found": {
            "message": "El servicio {domain}.{service} ha llamado al servicio {child_domain}.{child_service} que no se ha encontrado."
        }
    }
}